packages/web/src/components/search/DataSearch.js

```diff
@@ -238,18 +238,28 @@ class DataSearch extends Component {
 	};
 
 	onSuggestions = (results) => {
-		if (this.props.onSuggestion) {
-			return results.map(suggestion => this.props.onSuggestion(suggestion));
+		const { onSuggestion, renderSuggestion } = this.props;
+		if (onSuggestion) {
+			if (process.env.NODE_ENV !== 'production') {
+				console.warn('onSuggestion prop is deprecated please use renderSuggestion');
+			}
+			return results.map(suggestion => onSuggestion(suggestion));
```

Comment:
Why are we not parsing suggestions for `onSuggestion`? Is it to avoid breaking the existing usage?

Reply:
Yeah, also since this is a breaking change we can use `renderSuggestion` which would receive parsed suggestions.